apps/wpcom-block-editor/src/wpcom/features/test/redirect-onboarding-user-after-publishing-post.test.js

```diff
@@ -44,7 +44,11 @@ jest.mock( '@wordpress/data', () => ( {
 	},
 } ) );
 
-describe( 'RedirectOnboardingUserAfterPublishingPost', () => {
+/**
+ * This test file haven't been maintained and is failing.
```

Comment:
I think this test was forgotten because it didn't run on CI.
https://github.com/Automattic/wp-calypso/commits/8d2d52c00083119d1c2d53a37bb5e82cdba80e8c/apps/wpcom-block-editor/src/wpcom/features/test/redirect-onboarding-user-after-publishing-post.test.js